packages/flutter/lib/src/rendering/table.dart

```diff
@@ -402,6 +476,8 @@ class RenderTable extends RenderBox {
     assert(rows == null || children == null);
     assert(defaultColumnWidth != null);
     _columns = columns ?? (children != null && children.length > 0 ? children.first.length : 0);
+    // If [rows] is non-zero, won't we get into trouble when we call [addRow]
+    // below, which increments _rows for each sublist in [children]?
```

Comment:
children can't be non-null if children is non-null. One or the other (or both) must be null.